core/src/robot/Robot.cc

```diff
@@ -1485,10 +1487,26 @@ namespace jiminy
             }
         }
 
+        // Define robot notification method, responsible for updating the actuated joints
+        auto notifyRobot = [robot_=std::weak_ptr<Robot>(shared_from_this())](AbstractTransmissionBase & /*transmissionIn*/)
+            {
+                // Make sure the robot still exists
+                auto robot = robot_.lock();
+                if (!robot)
+                {
+                    PRINT_ERROR("Robot has been deleted. Impossible to notify transmission update.");
+                    return hresult_t::ERROR_GENERIC;
+                }
+
+                robot->refreshTransmissionProxies();
```

Comment:
Update the list of actuated joint here directly.

Reply:
this is inside a lambda function; so if i want to do this i think i need to put both `actuatedJointNames_` and `transmissionsHolder_` in the capture list. I can do so no problem, just be using the robot->refreshTransmission method it was nice and compact, what do you think?

Reply:
what the capture list and parameters it is not working; i would need to specify also the type `transmissionHolder_t` ect  in the transmissions.. I think the easiest would be to have a robot->updateActuadtedJoints function and there inside th same that I'm doing right now in the refresh transmissions

Reply:
I think better not to use `refreshTransmission` to clarify that the lambda is only about refreshing `actuatedJointNames_`. It is not more compacrt to call `refreshTransmission`, since actually the lambda is a method itself (in local scope but still). So you compare
```
def A:
   // Do something
   // Do something else
   
def B:
   // Call A
```
with 
```
def A:
   // Do something
   
def B:
   // Do something else
```

For me the second one is better.

Reply:
If should be working with the lambda. It is exactly the same as a method so there is no reason it would be any different. Just in comment the lambda you are trying to implement. It should be straightforward.